kits/arm/arm.py

```diff
@@ -18,10 +18,71 @@
 
 class ArmParams(object):
     
-    def __init__(self, family, moduleNames, hrdf):
+    def __init__(self, family, moduleNames, hrdf, gripperName=None):
         self.family = family
         self.hrdf = hrdf
         self.moduleNames = moduleNames
+        self.gripperName = gripperName
+        if self.gripperName is not None:
+            self.hasGripper = True
+        else:
+            self.hasGripper = False
+            
+            
+
+class Gripper(object):
+
+    def __init__(self, family, gripperName):
+        self.family = family
+        self.name = gripperName
+
+        self.grp = hebi.Lookup().get_group_from_names([family], [gripperName])
```

Comment:
for short variable names, prefer without abbreviations.  I know `grp` is a group here, but reading elsewhere in the code someone might assume it means `gripper`.

I'm ok with `cmd` below, fwiw, but wouldn't be opposed to writing that out fully too